tests/test_packages/test_connections/test_soef/test_soef.py

```diff
@@ -596,11 +602,73 @@ async def test_register_agent(self):
             assert self.connection.channel._ping_periodic_task is not None
 
     @pytest.mark.asyncio
-    async def test_request(self):
-        """Test internal method request_text."""
-        with patch("aea.helpers.http_requests.request"):
+    async def test_check_server_reachable(self):
```

Comment:
We also need an actual integration test for this. I.e. just check soef is reachable.